modules/search/instant-search/components/search-box.scss

```diff
@@ -10,25 +10,47 @@
 }
 
 input.jetpack-instant-search__box-input.search-field {
-	background: #fff;
 	border-radius: 5px;
 	border-style: solid;
 	border-width: 2px;
-	border-color: #b8b8b8;
-	color: black;
 	font-size: 18px;
 	line-height: 1;
 	margin: 0;
 	padding: 4px 14px;
 	text-indent: 32px;
 	vertical-align: middle;
+	transition: all 0.15s ease-in-out;
```

Comment:
This is being overridden by [this line](https://github.com/Automattic/jetpack/blob/update/instant-search-dark-theme-input-styles/modules/search/instant-search/components/search-box.scss#L184), which probably needs to be updated to:

```scss
transition: color 0.25s ease-in-out, border-color 0.25s ease-in-out;
```

Reply:
Good catch, I'll make that change @jsnmoon !

Reply:
Pushed a change in https://github.com/Automattic/jetpack/pull/14915/commits/d33911ddc8c506c874bcfac452893a6f0afb5e73